freqtrade/strategy/interface.py

```diff
@@ -599,18 +598,21 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
 
     def stop_loss_reached(self, current_rate: float, trade: Trade,
                           current_time: datetime, current_profit: float,
-                          force_stoploss: float, high: float = None) -> SellCheckTuple:
+                          force_stoploss: float, low: float = None,
+                          high: float = None) -> SellCheckTuple:
```

Comment:
While moving low into this method seems to complicate things - i think it makes the intend clearer - as it's clearly visible where we're using which rate.

This also slightly changes the input to "custom_stoploss" by using "open" in backtesting instead of low as current-rate ...but that's kinda intentional...